orttraining/orttraining/test/python/orttraining_test_checkpoint.py

```diff
@@ -13,7 +13,7 @@ def _single_run(execution_file, scenario, checkopint_dir):
     assert subprocess.call([sys.executable, execution_file, '--scenario', scenario, '--checkpoint_dir', checkopint_dir]) == 0
 
 def _distributed_run(execution_file, scenario, checkopint_dir):
-    assert subprocess.call(['mpirun', '-n', str(ngpus), '-x', 'NCCL_DEBUG=INFO', sys.executable, execution_file, '--scenario', scenario,  '--checkpoint_dir', checkopint_dir]) == 0
+    assert subprocess.call(['mpirun', '-n', str(ngpus), '--allow-run-as-root', '-x', 'NCCL_DEBUG=INFO', sys.executable, execution_file, '--scenario', scenario,  '--checkpoint_dir', checkopint_dir]) == 0
```

Comment:
why '--allow-run-as-root' is needed? root is a bad bad boy

Reply:
ah i added this to test in my docker env which always ran it as root. will delete it